### ISSUE TITLE
List supported languages

### ISSUE THREAD

**Issue by ostrolucky** [ast-grep/ast-grep#435]
You don't seem to list languages you support. I tried PHP and even though your online playground seem to support it, it doesn't find any matches. For example https://ast-grep.github.io/playground.html#eyJtb2RlIjoiUGF0Y2giLCJsYW5nIjoicGhwIiwicXVlcnkiOiIkQSgpOyIsImNvbmZpZyI6IiMgQ29uZmlndXJlIFJ1bGUgaW4gWUFNTFxucnVsZTpcbiAgYW55OlxuICAgIC0gcGF0dGVybjogaWYgKGZhbHNlKSB7ICQkJCB9XG4gICAgLSBwYXR0ZXJuOiBpZiAodHJ1ZSkgeyAkJCQgfVxuY29uc3RyYWludHM6XG4gICMgTUVUQV9WQVI6IHBhdHRlcm4iLCJzb3VyY2UiOiI8P3BocFxuXG5mdW5jdGlvbiBiYXIoKSB7XG4gIGJheigpO1xufSJ9

**Comment by HerringtonDarkholme** [comment]
No. PHP is not supported officially. https://ast-grep.github.io/guide/introduction.html#supported-languages

Contribution is welcome if you want to use PHP. 
https://github.com/ast-grep/ast-grep/blob/main/crates/language/src/lib.rs
https://github.com/ast-grep/ast-grep.github.io/blob/main/src/wasm_lang.rs#L27
